src/NzbDrone.Core/NzbDrone.Core.csproj

```diff
@@ -970,6 +970,10 @@
       <Link>MediaInfo.dll</Link>
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
+    <Content Include="..\MediaInfoDotNet.dll.config">
+      <Link>MediaInfoDotNet.dll.config</Link>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
```

Comment:
Note that this is currently done here: https://github.com/Sonarr/Sonarr/blob/develop/build.ps1#L117